.changelog.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why not just using default bump rules?

Reply:
Because with the latest changes I removed a whole action input `semantic-version` because it's no longer needed as the tool computes the next version by itself.

Now, by default only `Deprecated` change types would trigger a major version bump, and in the case of actions in general, and in my current case the removal of action inputs would cause a breaking change in the current workflows so new major version is needed.

I think that somewhat makes sense